### ISSUE TITLE
OffPolicyAlgorithm  and obs dict

### ISSUE THREAD

**Issue by hpugq** [RLE-Foundation/RLeXplore#15]
Is the OffPolicyAlgorithm now supported, as well as the observation space supported as dict?

**Comment by yuanmingqi** [comment]
yes, OffPolicyAlgorithm is supported. for dict observation space, pls refer to this issue https://github.com/RLE-Foundation/rllte/issues/44#issuecomment-1977575481
